### PR TITLE
refactor: improved test, make feature init ready for reuse

### PR DESCRIPTION
**Pull request by olehrakwix**
- make `setupRunningNodeEnv` more generic and move it from test to testkit
- fixed `setupRunningNodeEnv` to run real feature discovery instead of having it hardcoded
- prepare `setupRunningNodeEnv` for reuse
- moved `test-project` from package to `test-fixtures`

### CODE REVIEW

**Review by cylon3035 — APPROVED** [review]

File: packages/electron-commons/test/initializers/electron-node.spec.ts [hunk 1]
```diff
@@ -1,60 +1,31 @@
...
 
-const setupRunningEnv = async ({
-    errorMode,
-    handleUncaught,
-    stdio = 'ignore',
-}: {
-    errorMode?: 'exception' | 'exit' | 'promiseReject' | 'out-of-memory' | 'no-error';
-    handleUncaught?: boolean;
+interface SetupRunningFeatureOptions {
+    config: Partial<ErrorTypeConfig>;
```
Comment (alisey) [inline packages/electron-commons/test/initializers/electron-node.spec.ts:16]: Options object that contains a config object? Please no.

Comment (olehrakwix) [reply packages/electron-commons/test/initializers/electron-node.spec.ts:16]: `options` - the options of environment initialization
`config` - config of feature that is being initialized

Comment (alisey) [reply packages/electron-commons/test/initializers/electron-node.spec.ts:16]: Then call it featureConfig. Isn't it weird that feature config has type `ErrorTypeConfig`?

Comment (alisey) [reply packages/electron-commons/test/initializers/electron-node.spec.ts:16]: Also, `errorMode` in feature config? Are we taking about the same feature config?
https://github.com/wixplosives/engine/blob/master/packages/core/src/entities/config.ts#L9

**Pull request merged**